verify/src/method/v29.rs

```diff
@@ -0,0 +1,196 @@
+    Method::new_modelled("getmempoolentry", "GetMempoolEntry", "get_mempool_entry"),
+    Method::new_modelled("getmempoolinfo", "GetMempoolInfo", "get_mempool_info"),
+    Method::new_modelled("getrawmempool", "GetRawMempool", "get_raw_mempool"),
+    Method::new_modelled("gettxout", "GetTxOut", "get_tx_out"),
+    Method::new_string("gettxoutproof", "get_tx_out_proof"),
+    Method::new_modelled("gettxoutsetinfo", "GetTxOutSetInfo", "get_tx_out_set_info"),
+    Method::new_modelled("gettxspendingprevout", "GetTxSpendingPrevout", "get_tx_spending_prevout"),
+    Method::new_modelled("importmempool", "ImportMempool", "import_mempool"),
+    Method::new_no_model("loadtxoutset", "LoadTxOutSet", "load_tx_out_set"),
+    Method::new_nothing("preciousblock", "precious_block"),
+    Method::new_no_model("pruneblockchain", "PruneBlockchain", "prune_blockchain"),
+    Method::new_no_model("savemempool", "SaveMempool", "save_mempool"),
+    Method::new_modelled("scanblocks", "ScanBlocks", "scan_blocks"),
+    Method::new_modelled("scantxoutset", "ScanTxOutSet", "scan_tx_out_set"),
+    Method::new_no_model("verifychain", "VerifyChain", "verify_chain"),
+    Method::new_modelled("verifytxoutproof", "VerifyTxOutProof", "verify_tx_out_proof"),
+    // control
+    Method::new_no_model("getrpcinfo", "GetRpcInfo", "get_rpc_info"),
+    Method::new_no_model("getmemoryinfo", "GetMemoryInfoStats", "get_memory_info"),
+    Method::new_string("help", "help"),
+    Method::new_no_model("logging", "Logging", "logging"),
+    Method::new_nothing("stop", "stop"),
+    Method::new_numeric("uptime", "uptime"),
+    // mining
+    Method::new_modelled("getblocktemplate", "GetBlockTemplate", "get_block_template"),
+    Method::new_no_model("getmininginfo", "GetMiningInfo", "get_mining_info"),
+    Method::new_nothing("getnetworkhashps", "get_network_hashes_per_second"),
+    Method::new_modelled("getprioritisedtransactions", "GetPrioritisedTransactions", "get_prioritised_transactions"),
+    Method::new_bool("prioritisetransaction", "prioritise_transaction"),
+    Method::new_nothing("submitblock", "submit_block"),
+    Method::new_bool("prioritisetransaction", "prioritise_transaction"),
+    Method::new_nothing("submitblock", "submit_block"),
+    Method::new_nothing("submitheader", "submit_header"),
+    // network
+    Method::new_nothing("addnode", "add_node"),
+    Method::new_nothing("clearbanned", "clear_banned"),
+    Method::new_nothing("disconnectnode", "disconnect_node"),
+    Method::new_no_model("getaddednodeinfo", "GetAddedNodeInfo", "get_added_node_info"),
+    Method::new_no_model("getaddrmaninfo", "GetAddrManInfo", "get_addr_man_info"),
+    Method::new_numeric("getconnectioncount", "get_connection_count"),
+    Method::new_no_model("getnettotals", "GetNetTotals", "get_net_totals"),
+    Method::new_modelled("getnetworkinfo", "GetNetworkInfo", "get_network_info"),
+    Method::new_no_model("getnodeaddresses", "GetNodeAddresses", "get_node_addresses"),
```

Comment:
```suggestion
    Method::new_modelled("getnodeaddresses", "GetNodeAddresses", "get_node_addresses"),
```

Recent change in master needs to be added.

Reply:
Niiiice -done.

Reply:
I fixed up all the `mod.rs` in `types` too.